lib/views/pages/index.html

```diff
@@ -27,3 +27,28 @@
         </figure>
     </article>
 </section>
+
+<section class="layout-section" id="newsletter">
+    <article class="article newsletter">
+        <h2>News<span class="special-color">letter</span></h2>
+
+        <p class="description">Subscribe to the NightlyBuild newsletter to receive updates from your favorite conference owls.</p>
+
+        <!-- Begin MailChimp Signup Form -->
+        <div class="newsletter-form" id="mc_embed_signup" data-fx-fade>
+            <form action="//drublic.us9.list-manage.com/subscribe/post" method="get" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate form" target="_blank" novalidate>
```

Comment:
Breaks locally and can also enable phishing attacks due to missing HSTS and HPKP on our side. ;) But yeah, not one of our main concerns probably.